backend/modules/parsers/multimodalparser.py

```diff
@@ -61,29 +61,27 @@ class MultiModalParser(BaseParser):
 
     def __init__(
         self,
-        additional_config: dict = None,
         *args,
         **kwargs,
```

Comment:
same comment as here: https://github.com/truefoundry/cognita/pull/286/files?show-viewed-files=true&file-filters%5B%5D=#r1693115703

Reply:
The link does not open but if the comment is regarding max_chunk_size, this is not used in `MultiModalParser`